.environ/.env.dev

```diff
@@ -30,8 +30,19 @@ ENABLE_GOOGLE=true
 # You'll get this email from your credentials file; it's the email of the service account
 # used to authenticate to the Google Drive API.
 GOOGLE_AUTH_CLIENT_SERVICE_EMAIL='bot-name@project-name.iam.gserviceaccount.com'
+
+# This is the ID of the Google Drive folder that will contain the directory structure
+# E.g. GOOGLE_DRIVE_ROOT='1NLXlnhTY4RcpGQuvFgRNPiv0fYZ3peI7'
+# Check the directory structure and sharing requirements in README
 GOOGLE_DRIVE_ROOT='root-folder-id'
 
+# Name of the .p12 file downloaded from Google Cloud. It can be stored in the root directory.
+GOOGLE_AUTH_PKCS12_FILE_PATH='authorization-file-downloaded-from-google.p12'
+
+# The connection should work when you keep this user e-mail with the default value '', and also if
+# you use the same e-mail used for the service account.
+# GOOGLE_AUTH_CLIENT_SERVICE_EMAIL=''
```

Comment:
This is already covered above BTW, in line 32 at the moment. I also don't recall if it's needed to be set, but let's leave it being set for now, since I'm not 100% certain myself yet if it needs to or not be and better just be on the safe side.

Reply:
Ah, I think you meant the `USER` email, not the service one.

Reply:
Yes, it was a typo. Fixed.